pop_tools/xgcm_util.py

```diff
@@ -60,6 +60,7 @@ def _label_coord_grid_locs(ds):
         'DZU': '3221',
         'DZT': '3111',
         'HT': '2110',
+        'DZBC': '2110',
```

Comment:
@ALDepp is this right?

Reply:
looks right to me.

Reply:
:+1: thanks!

Reply:
I think this is correct. This is the rubric:
```
grid_loc 1: dimensionality
grid_loc 2: horizontal location in x: 1 = TLONG; 2 = ULONG
grid_loc 3: horizontal location in y: 1 = TLAT; 2 = ULAT
grid_loc 4: vertical grid: 0 = surface, 1 = z_t, 2 = z_w, 3 = z_w_bot, 4 = z_t_150m
```
which should be in documentation somewhere!